core/tests/puppeteer-acceptance-tests/spec/translation-admin-tests/remove-translation-rights.spec.ts

```diff
@@ -65,6 +66,10 @@ describe('Translation Admin', function () {
     DEFAULT_SPEC_TIMEOUT
   );
 
+  afterEach(function () {
```

Comment:
Probably not the best since you might want to exclude some errors so its better to be explicit (e.g. Blog Post Title Duplicate).

Reply:
But then, could you have each of this contain only the list of the errors to be excluded, and have the generic logic in the superclass?

Reply:
Might be able to only way I can think of is by adding it to spec done inside the reporter but kind of hard to pass through parameters there and there isn't any simple way of doing this that I know of in jasmine.

Reply:
BTW changed it so this functionality is implemented in reporter still fails when there are console errors and is better since there isn't really any "stack".

Reply:
^ Disregard this, I just implemented it so console errors are reported in the spec done instead of after each.